view/rooms/new/FormController.js

```diff
@@ -45,6 +45,7 @@ class FormController extends React.Component {
     this.getTypes();
   }
 
+  // Gets the room types that are available for use
   getTypes(){
```

Comment:
Would be possible to change the method name to getAvailableRoomType and then remove the comment? Comments are really easy to be outdated.